internal/dao/bootstrap.go

```diff
@@ -147,7 +147,7 @@ func RegisterBootstrapAgent(ctx context.Context, db bun.Tx, token string) error
 	case sentry.BootstrapAgentState_NotRegistered.String():
 		ba.TokenState = sentry.BootstrapAgentState_Approved.String()
 	case sentry.BootstrapAgentState_NotApproved.String(), sentry.BootstrapAgentState_Approved.String():
-		if !bat.IgnoreMultipleRegister {
+		if !bat.IgnoreMultipleRegister || ba.Fingerprint != fingerprint {
```

Comment:
@niravparikh05 How about we differentiate not set IgnoreMultipleRegister and fingerprint mismatch errors, and return appropriate error instead? If server return `fingerprint mismatch` then user simply understand the bootstrap yaml is already used which is the sole purpose of fingerprint as of now. Whereas other means multiple register is dismissed by server because IgnoreMultipleRegister is not set.